src/System.Runtime/tests/System/ExceptionTests.cs

```diff
@@ -4,69 +4,37 @@
 
 using System;
 using System.IO;
+using System.Reflection;
 using Xunit;
 
 namespace System.Tests
 {
-    public static class NullReferenceExceptionTests
+    public static class ExceptionTests
     {
-        private const string exceptionMessage = "Created an exception";
-        private const string innerExceptionMessage = "Created an InnerException";
-        private const uint COR_E_ENDOFSTREAM = 0x80070026;
-        private const uint E_POINTER = 0x80004003;
-
-        [Fact]
-        public static void NullReferenceException_Ctor_Empty()
-        {
-            NullReferenceException i = new NullReferenceException();
-            Assert.Equal(E_POINTER, (uint)i.HResult);
-        }
-
-        [Fact]
-        public static void NullReferenceException_Ctor_String()
-        {
-            NullReferenceException i = new NullReferenceException(exceptionMessage);
-            Assert.Equal(exceptionMessage, i.Message);
-            Assert.Equal(E_POINTER, (uint)i.HResult);
-        }
-
         [Fact]
-        public static void NullReferenceException_Ctor_String_Exception()
+        public static void Exception_GetType()
         {
-            Exception ex = new Exception(innerExceptionMessage);
-            NullReferenceException i = new NullReferenceException(exceptionMessage, ex);
-
-            Assert.Equal(exceptionMessage, i.Message);
-            Assert.Equal(i.InnerException.Message, innerExceptionMessage);
-            Assert.Equal(i.InnerException.HResult, ex.HResult);
-            Assert.Equal(E_POINTER, (uint)i.HResult);
+            Assert.Equal(typeof(Exception), (new Exception()).GetType());
+            Assert.Equal(typeof(NullReferenceException), (new NullReferenceException()).GetType());
         }
 
         [Fact]
-        public static void EndOfStreamException_Ctor_Empty()
+        public static void Exception_TargetSite()
```

Comment:
In other projects, we've separated out the tests for netstandard1.7... have we done that with System.Runtime?